Automattic/jetpack
# My Jetpack: Fix Protect card Tooltip placement & content issues

This PR fixes a few minor issues with the placement and content of the Tooltips in the Protect product card in My Jetpack.





Original issues (screenshots):

Mobile view - Tooltip issues
Threats Tooltip | Auto-firewall Tooltip (Mobile view)
--- | ---
![protect-before](https://github.com/user-attachments/assets/9f98218b-90b2-4077-84f9-41059a87e525) | ![Screen Shot on 2024-12-19 at 16-04-06](https://github.com/user-attachments/assets/e6af241a-8c39-4348-8d2e-fbda97f310cd)

** Notice in the "Auto-fix threats" tooltip (left img), it says "The last scan identified 6 critical threats.. But..."; **This is wrong**, as you can see, there is 6 (regular) threats and 1 critical threat.
This PR also fixes this incorrect wording ⤴️.

Fixes https://github.com/Automattic/jetpack/issues/40433

## Proposed changes:

* Fixes the appearance "Auto-fix threats" tooltip (increases the width to better fit the content)
* Fixes the wording/content in the "Auto-fix threats" tooltip with the correct distinction between the number of (regular) threats and critical threats. Also adjusts wording to make it a little more clear that not all threats can be auto-fixed.
* Fixes the "Auto-firewall" tooltip, on mobile so that half of it is no longer overflowing off the side of the viewport.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Install/activate Jetpack Protect with a Scan upgrade
- Using the Scan Helper in the Jetpack Debug Helper/Tools plugin, select the first EIACAR Threats selection to Add five EICAR threats to the site (one for every severity).
- Run a scan to detect them
- Proceed to the My Jetpack section, scroll down to the Protect product card.
- Hover over and click the "Threats" info icon tooltip to view the tooltip. It should look like this screenshot:

Desktop | Mobile
--- | ---
![Screen Shot on 2024-12-20 at 02-01-38](https://github.com/user-attachments/assets/15b1d89e-38f6-4918-b852-579db2b783a7) | ![Screen Shot on 2024-12-20 at 02-02-35](https://github.com/user-attachments/assets/0edd176d-4266-40a6-8b26-0e6f0eb88b3e)

- In mobile view Hover over and click the "Auto-firewall" info icon tooltip. It should look like this screenshot:

![Screen Shot on 2024-12-19 at 14-06-38](https://github.com/user-attachments/assets/613aeb7f-bcdf-4e9e-ba9f-41fb20d48cd9)
- Thats it!
